tests/console/hwloc_testsuite.pm

```diff
@@ -0,0 +1,50 @@
+# SUSE's hwloc tests
+#
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Run testsuite included in hwloc sources
+# Maintainer: Thomas Blume <tblume@suse.com>
+
+use base "consoletest";
+use warnings;
+use strict;
+use testapi;
+use utils;
+
+sub run() {
+    # install hwloc testsuite
+    become_root();
```

Comment:
Hm, I'm not in a root shell at the start of the test. Without become_root(), the test fails see: http://emerson/tests/176#step/hwloc_testsuite/1
Is this probably caused by skipping the other tests before hwloc_testsuite?